downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,454 @@
+| `getKeyType`                                                              | `io.ballerina.runtime.api.values.BTable`           |
+| `getDescribingType`                                                       | `io.ballerina.runtime.api.values.BTypedesc`        |
+| `getType`                                                                 | `io.ballerina.runtime.api.values.BValue`           |
+| `getType`                                                                 | `io.ballerina.runtime.api.utils.TypeUtils`         |
+
+For example, if an intersection type is defined in the following way,
+
+```ballerina
+type ReadonlyIntArray readonly & int[];
+
+ReadonlyIntArray array = [1, 2, 3, 4];
+```
+
+the results of the runtime API calls will be as follows.
+
+| **Runtime API call**                                   | **Result**                                                     |
+|--------------------------------------------------------|----------------------------------------------------------------|
+| `array.getType()`                                      | This will return an `IntersectionType` instead of `ArrayType`. |
+| `getDescribingType()` on `ReadonlyIntArray` type       | This will return an `IntersectionType` instead of `ArrayType`. |
+| `getImpliedType()` on received `IntersectionType` type | This will return an `ArrayType`.                               |      
+
+### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.8.0+label%3ATeam%2FjBallerina+label%3AType%2FBug+is%3Aclosed).
+
+## Standard library updates
+
+### New features
+
+#### `mqtt` package
+- Introduced the `mqtt` standard library package, which provides an implementation to interact with message brokers using the MQTT protocol.
+
+#### `java.jms` package
+- Introduced the `java.jms` standard library package, which provides an implementation to interact with message brokers using the JMS protocol.
+
+#### `graphql` package
+
+- Introduced the `DataLoader` functionality.
+
+### Deprecations
+
+#### `graphql` package
+
+- Deprecated the `executeWithType()` method of the `graphql:Client`.
+
+### Improvements
+
+#### `graphql` package
+
+- Added support for the `@deprecated` directive to output objects defined using record types.
+- Added support for printing the GraphiQL URL to `stdout`.
+- Added support for generating a subgraph SDL schema at compile time.
+
+#### `log` package
+- Made adjustments to the behavior of the log print functions when an error object is provided for the `error` key. In the previous version, it would only display the error message. However, in the updated version, it now provides more detailed information, including both the error message and the stack trace for the current error, as well as any underlying errors that contributed to the current error.
```

Comment:
```suggestion
#### `log` package

- Made adjustments to the behavior of the log print functions when an error object is provided for the `error` key. In the previous version, it would only display the error message. However, in the updated version, it now provides more detailed information, including both the error message and the stack trace for the current error, as well as any underlying errors that contributed to the current error.
```